components/portal/widgets/partials/widget-card.html

```diff
@@ -26,8 +26,8 @@
       <p class="overlay-warning-icon">
         <md-icon class="md-warn">warning</md-icon>
       </p>
-      <ng-show="widget.maintenanceMessage"><p>{{ widget.maintenanceMessage }}</p></ng-show>
-      <ng-hide="widget.maintenanceMessage"><p>This app is undergoing maintenance and is temporarily available.</p></ng-hide>
+      <ng-show="widget.widgetConfig.maintenanceMessage"><p>{{ widget.maintenanceMessage }}</p></ng-show>
```

Comment:
should `{{ widget.maintenanceMessage }}` be `{{ widget.widgetConfig.maintenanceMessage }}`?
Or are those different properties?